FontLoader.js

```diff
@@ -661,8 +745,8 @@
 					return;
 				}
 			}
-			
-			if (!this._continuous) {
+
+			if (!this._continuous || this._delegate.isFinished()) {
```

Comment:
No need for adding isFinished check.
When FontLoader finishes, it removes the container element from DOM, so no `scroll` events will be fired.